server/api/swagger/user.yaml

```diff
@@ -547,6 +547,51 @@ paths:
         - bearer
       x-security-scopes:
         - user
+  /user/test/get-otp:
```

Comment:
the format starting with user is not good. these should start with /test/ to be really different. now its getting mixed up with user endpoints. Also we should put these into a nice test.yaml file and not mix them up on swagger with user endpoints